README.md

```diff
@@ -1,3 +1,4 @@
 # PR_test
 
 Some text. 
+Some other text.
```

Comment:
Adding a "single comment" comment on code (lines 3 and 4)

Reply:
Replying to 'single comment' addition mentioning code change.